conf/messages.en

```diff
@@ -626,3 +626,23 @@ companyRecipientName.heading = What is the recipient’s company name?
 companyRecipientName.error.required = Enter the recipient’s company name
 companyRecipientName.error.invalid.characters = Recipient’s company name must only include letters, numbers, and special characters such as hyphens, spaces and apostrophes
 companyRecipientName.error.length = Recipient’s company name must be 160 characters or fewer
+
+isMemberOrConnectedParty.title = Is the recipient a member or a connected party?
```

Comment:
Are these under `isMemberOrConnectedParty` used anywhere?

Reply:
I think there was an issues when I rebased as I was jumping between the 2 stories (this one and PSR-215) - I'm going to merge PSR-215 after this so I can sort out this issue then